cwms-data-api/src/main/java/cwms/cda/data/dto/Office.java

```diff
@@ -19,10 +19,12 @@
 @Schema(description = "A representation of a CWMS office")
 @JsonRootName("office")
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
-@FormattableWith(contentType = Formats.XML, formatter = XMLv1.class)
-@FormattableWith(contentType = Formats.XMLV2, formatter = XMLv2.class)
-@FormattableWith(contentType = Formats.JSON, formatter = JsonV1.class)
-@FormattableWith(contentType = Formats.JSONV2, formatter = JsonV2.class)
+@FormattableWith(contentType = Formats.XML, formatter = XMLv2.class)
+@FormattableWith(contentType = Formats.XMLV1, formatter = XMLv1.class)
+@FormattableWith(contentType = Formats.XMLV2, formatter = XMLv2.class, aliases = {Formats.XML})
+@FormattableWith(contentType = Formats.JSON, formatter = JsonV2.class)
```

Comment:
If JSON is an alias for JsonV2 do you need this line?

Reply:
Ah, good point, don't need that, or XML